tools/packaging/static-build/initramfs/build.sh

```diff
@@ -33,14 +32,18 @@ package_output_dir="${package_output_dir:-}"
 [ -n "${lvm2_repo}" ] || die "Failed to get lvm2 repo"
 [ -n "${lvm2_version}" ] || die "Failed to get lvm2 version"
 
-sudo docker build \
-	-t "${container_image}" "${script_dir}"
+container_image="${CC_BUILDER_REGISTRY}:initramfs-cryptsetup-${cryptsetup_version}-lvm2-${lvm2_version}-$(get_last_modification ${repo_root_dir} ${script_dir})"
```

Comment:
`CC_BUILDER_REGISTRY` is not set anywhere

Reply:
Good spot - thank you I thought I'd caught all of those, but obvious missed one! Fixing now.